### ISSUE TITLE
Silent's Mechanisms compatability issue

### ISSUE THREAD

**Issue by Arisuterra** [Sinhika/SimpleOres2#50]
hello, I been experimenting with your mods
Simple Ores
Netherocks
Fusion
Aesthetics
with a modlist of about 22, and the most recent update made them unable to recognize (first three not aesthetics) silent's mechanisms.
I been downloading and redownloading, trying to edit but unable to put them back in which they are recognized by the systems. It seems the mod meta is listing the mods version number without the full set (1.16.3-1.0.1 instead of 1.16.3-1.0.1+77) could an updated version of simple ores netherrocks and fusion be put out with the fix on that line? I understand im asking quite a mundane thing but i cant go back to not using your wonderful mods with silent's they make the perfect game and i just got the create mod along side it. moxxicraft server will have some fun rpg dungeons when it all comes together.
If im doing something wrong and its so simple i apologize but appreciate you fixxin meh, i search over the internet for 2 days trying to set things back up so im sure that wont be the case but let me know if theres more info i should provide.

**Comment by Sinhika** [comment]
Which version did the problem appear with? ("Most recent" isn't helpful when I'm tossing up alpha versions every other day). If it's one of the alphas or the recent beta version, they have NOT been tested with Silent's stuff. I'm trying to get a broad swath of fixes and upgrades done and move to a new release set for all the mods; that set of course will be tested with Silent's Gear & Mechanisms. 

Or did a new version of Silent's break compatibility with the old release version of Simple Ores & friends that used to work with it? I'm not clear here.

**Comment by Arisuterra** [comment]
it was all the beta's about 3 days ago
versions
simple ores 2-1.16.5-2.4.6.2
netherrocks 1.16.5-2.3.5.2
fusion 1.16.5-2.7.3.3
and it may be silent's but i looked into all 3 with visual studio code and found the compatibilities and in mechanisms its just off a bit because of how long the version is. and to paint the whole picture i was cleaning house shortening the modlist (love akkamads stuff but making a datapack to implement hadite, sterling and black silver is a bit much for me) saw some of the mods were updated so i downloaded them all and prepared to setup my world (663436168) its a nice seed. but never got to the menu as all three mentioned showed that they were incompatible with a different version but even when i rolled back mechanism the extended name kept it from being recognized by the very version it was. and of coarse i tried the latter with them without dropping below minecraft 1.16.5
it seems to be the naming thats causing all the conflicts.
for now ive dropped mechanisms to enjoy experimenting in creative mode 
thank you for giving me a reply so quickly, sorry the reply is so long.

**Comment by Sinhika** [comment]
I've figured out how to specify the necessary versioning in mods.toml by trial and error. (Forge could really use better documentation). SimpleOres2 is fixed, the others should follow later today.

Fixed with SimpleOres2 1.16.5-2.4.6.6

**Comment by Sinhika** [comment]
Fixed the rest:
Netherrocks 1.16.5-2.3.5.4
Fusion 1.16.5-2.7.3.6 (or later)
Aesthetics and Machines were not affected, as they did not have dependencies on Silent's Mechanisms.

**Comment by Arisuterra** [comment]
Wow you guys are amazing, it's all working I have tested it a bit gonna put it through the tests on my server. already I have noticed Illumenite not granting kitty-vision on helms, bracelets and rings but that might have been there before i do remember struggling with it before all the updates. but I wanna work with that more and see what's up before coming back here plus its more likely silent's mods at fault in this case or i'm not getting it which is most likely xp but seriously you guys rock I meant to get back and test it out and get back to you sooner but i had to work the other day. and I was not able to copy the transaction id or anything but i did give a lil donation. thanks I look forward to what your team makes. :)
